docs/data-sources/identity_project_v3.md

```diff
@@ -14,6 +14,16 @@ data "opentelekomcloud_identity_project_v3" "project_1" {
 }
 ```
 
+### Query Current Project details
+
+If no `name` or `domain_id` are not provided, data source gets info about the project from client configuration.
```

Comment:
Double negative here: `If NO ... are NOT provided...`. 
Also, maybe better to say smth like `currently used project` instead of `the project from client configuration`.